src/test/java/it/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMIT.java

```diff
@@ -66,6 +69,43 @@ public void testCheckout() throws Exception {
         assertTrue(build.getWorkspace().child("add_file").isDirectory());
     }
 
+    @Test
+    public void testCheckoutAndPush() throws Exception {
+        project.setScm(createSCM("*/master"));
+        String uniqueMessage = UUID.randomUUID().toString();
+        Shell postScript = new Shell("git checkout master\n" +
```

Comment:
Slight preference for this script to go in a file in `resources` and be read in rather than using string concatenation